R/params-functions.R

```diff
@@ -12,7 +12,7 @@
 #' @export
 getParams <- function(params, names) {
 
-    checkmate::assertClass(params, classes = "Params")
+    #checkmate::assertClass(params, classes = "Params")
```

Comment:
I honestly don't remember exactly why I commented these out! I think it was because of issues I was having early on in splatPop development that I've since fixed. I've added them back and so far no problems...